oc-chef-pedant/spec/api/controls_spec.rb

```diff
@@ -12,419 +12,432 @@
-
-    context 'POST /controls' do
-      after :each do
-        delete(api_url("clients/client.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
-
-        delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
-      end
-
-      it 'returns a 201 ("Created") for client' do
-        post(api_url("/controls"),
-            client,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 201
-              })
-            end
-      end
-
-      it 'returns a 403 ("Forbidden") for normal_user' do
-        post(api_url("/controls"),
-            normaluser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 403
-              })
-            end
-      end
-
-      it 'returns a 403 ("Forbidden") for admin_user' do
-        post(api_url("/controls"),
-            adminuser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 403
-              })
-            end
-      end
-
-    end
-
-    context 'GET /controls' do
-      after :each do
-        delete(api_url("clients/client.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
-
-        delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
-
-      end
-
-      it 'returns a 405 ("Forbidden") for client' do
-        get(api_url("/controls"),
-            client,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+      let(:client){ platform.create_client('client.opscode.piab') }
+      let(:clientwithnonode){ platform.create_client('clientwithnonode.opscode.piab') }
+      let(:normaluser){ normal_user }
+      let(:adminuser){ admin_user }
+
+      context 'POST /controls' do
+        after :each do
+          delete(api_url("clients/client.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
+
+          delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
+        end
 
-      it 'returns a 405 ("Method Not Allowed") for normal_user' do
-        get(api_url("/controls"),
-            normaluser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+        it "returns a 201 ('Created') for client" do
```

Comment:
I think this was an incidental change when we undid an earlier version of the updates.  Still, we'll keep it consistent with what's in the rest of the file.